src/components/tables/PrefixAlerts/index.tsx

```diff
@@ -65,6 +65,7 @@ function PrefixAlertTable() {
 
     return (
         <TableHydrator
+            disableQueryParamHack
```

Comment:
This table does open with navigation but still got messed up with the hack so just switching off.